### ISSUE TITLE
Where is the unified target 

### ISSUE THREAD

**Issue by sugaarK** [jhemcu/FC-ESC-Firmware#4]
@jhemcu  where is your unified target for 4.3? This repo has been open for months...  we are about to release 4.3 release and your new h7 AIO doesn’t have a working target..  I know people buying these things and they can’t use them with the current code base...

**Comment by blakemberry** [comment]
@sugaarK have you figure out a work around? Just put this board on my babytooth and am having alot of issues. IE only two motors spin up. All chime up when plugged into betaflight. If I disable rpm filtering all motors work properly. Any help with setup would be greatly appreciated.

**Comment by jhemcu** [comment]
特别说明，由于H743固件的原因，暂时不支持双向DSHOT！






董家良
津航电子


 联系我









&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jhemcu/FC-ESC-Firmware"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月5日(星期三) 下午5:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jhemcu/FC-ESC-Firmware] Where is the unified target  (Issue #4)





 
@sugaarK have you figure out a work around? Just put this board on my babytooth and am having alot of issues. IE only two motors spin up. All chime up when plugged into betaflight. If I disable rpm filtering all motors work properly. Any help with setup would be greatly appreciated.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by jhemcu** [comment]
特别说明，由于H743固件的原因，暂时不支持双向DSHOT！


你可以刷入&nbsp;&nbsp;IFLIGHT_H743_AIO 固件


目前是BF平台的H743对双向DSHOT的支持都不好。包含所有品牌，请知晓






董家良
津航电子


 联系我









&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jhemcu/FC-ESC-Firmware"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月31日(星期五) 晚上6:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;[jhemcu/FC-ESC-Firmware] Where is the unified target  (Issue #4)





 
@jhemcu  where is your unified target for 4.3? This repo has been open for months...  we are about to release 4.3 release and your new h7 AIO doesn’t have a working target..  I know people buying these things and they can’t use them with the current code base...
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by howels** [comment]
Bidirectional dshot issues have been resolved in current BetaFlight 4.3 code. I am running another board, the Matek H743 with bidirectional DShot. This is working correctly in BetaFlight 4.3.

Please submit an update so that JHEH743 can be used with latest BetaFlight 4.3.

I have helped earlier to create a unified target for your JHEF405-PRO, we would appreciate a unified target config from JHEMCU for JHEH743.  This looks like a fantastic board but it needs software for customers to buy it and enjoy the product.

**Comment by howels** [comment]
I got bidirectional dshot working on all 4 motors with dshot_bitbang, so looks like a timer config issue.

Edit - that appears to have been a chance event, now only motor 1 & 2 work reliably.

**Comment by sugaarK** [comment]
I agree @howels it the manufacturer needs a help with this we the Betaflight team can help. Also @jhemcu  know that after we finalise the 4.3 release we’ll be doing specifications for manufacturers to follow with new flight controllers so we can avoid these issues moving forward..  and questions @jhemcu  please email in English  at sugarK@betaflight.com
